### PR TITLE
Outbound plane to hk

### PR DESCRIPTION
**Pull request by paul-grassi**

### CODE REVIEW

File: sp800-63a/sec5_proofing.md [hunk 1]
```diff
@@ -118,20 +118,29 @@ The following requirements apply to the identity verification steps for IAL 2 an
 	- The CSP SHALL time out KBV sessions after 2 minutes of inactivity per question.  In cases of session timeout, the CSP SHALL restart the entire KBV process.
 
 
-### <a name="vip"></a>5.4.3. Virtual In-person Proofing Requirements
+### <a name="vip"></a>5.4.3. In-person Proofing Requirements
+
+#### 5.4.3.1 General Requirements
+
+1.	The CSP SHALL have the operator view the biometric source (e.g., fingers or face) for presence of non-natural materials and perform such inspections as part of the proofing process.
+2.	The CSP SHALL collect biometrics in such a way that ensures that the biometric is collected from the applicant, and not another individual. All biometric requirements in [SP 800-63B, Section 5.2.3 Biometric Considerations](sp800-63b.html/#biometric_use) apply.
```
Comment (jimfenton) [inline sp800-63a/sec5_proofing.md]: Suggest using "biometric performance requirements", as this biometric is not used with another factor. Or perhaps we should break -63B 5.2.3 into two parts.

File: sp800-63a/sec5_proofing.md [hunk 2]
```diff
@@ -118,20 +118,29 @@ The following requirements apply to the identity verification steps for IAL 2 an
...
+
+
+
+#### 5.4.3.2. Requirements for in-person proofing performed over remote channels
+
+It is possible for a CSP to achieve the security and confidence comparable to in-peson proofing over remote channels.  The following requirements establish comparability between in-person transactions where the enrollee in the same physical locatin as the CSP or when the enrollee is remote to the CSP.
```
Comment (jimfenton) [inline sp800-63a/sec5_proofing.md]: locatin -> location

File: sp800-63a/sec5_proofing.md [hunk 3]
```diff
@@ -118,20 +118,29 @@ The following requirements apply to the identity verification steps for IAL 2 an
...
-2. The CSP SHALL require all actions taken by the applicant during the enrollment and identity proofing process are visible.
-4. All digital verification of evidence (e.g., via chip or wireless technologies) SHALL be performed by scanners and sensors that are integrated into the solution and in the entire field of view of the camera and the remote, live operator.
-5. Collection of biometrics SHALL be done in such a way that ensures that the biometric is collected from the applicant, and not another individual. All biometric requirements in [SP 800-63B, Section 5.2.3 Biometric Considerations](sp800-63b.html/#biometric_use) apply.
+2. 2.	The CSP SHALL require all actions taken by the applicant during the enrollment and identity proofing process to an operator. are clearly visible and without doubt.
+4. The CSP SHALL provide that all digital verification of evidence (e.g., via chip or wireless technologies) is performed by scanners and sensors that are integrated into the solution and in the entire field of view of the camera and the remote, live operator.
```
Comment (jimfenton) [inline sp800-63a/sec5_proofing.md]: Suggest provide -> require. is -> be

File: sp800-63a/sec5_proofing.md [hunk 4]
```diff
@@ -118,20 +118,29 @@ The following requirements apply to the identity verification steps for IAL 2 an
...
-6. The CSP SHALL require operators to have undergone a training program to detect potential fraud.
-7. The CSP SHALL employ tamper detection and resistance features appropriate for the environment in which it is located. For example, a kiosk located in a restricted area or one where it is monitored by a trusted individual requires less tamper detection than one that is located in a semi-public area such as a retail store.
-8. All communications CSP SHALL take place over a mutually authenticated encrypted session.
+6. The CSP SHALL require operators to have undergone a training program to detect potential fraud and to properly perform a virtual in-process proofing session.
+7. The CSP SHALL employ physical tamper detection and resistance features appropriate for the environment in which it is located. For example, a kiosk located in a restricted area or one where it is monitored by a trusted individual requires less tamper detection than one that is located in a semi-public area such as a retail store.
+8. The CSP SHALL ensure all communications take place over a mutually authenticated encrypted session.
```
Comment (jimfenton) [inline sp800-63a/sec5_proofing.md]: Suggest "ensure" -> "ensure that"
Should it say, "all communications with the kiosk"?

File: sp800-63a/sec5_proofing.md (hunk 3 above)
Comment (jimfenton) [reply sp800-63a/sec5_proofing.md]: Also suggest solution -> kiosk because that's what we call it elsewhere.

File: sp800-63a/sec5_proofing.md [hunk 5]
```diff
@@ -118,20 +118,29 @@ The following requirements apply to the identity verification steps for IAL 2 an
...
 
 1. The CSP SHALL monitor the entire identity proofing transaction, from which the applicant SHALL NOT depart during the identity proofing session.  For example, by a continuous high-resolution video transmission of the applicant.
-2. The CSP SHALL require all actions taken by the applicant during the enrollment and identity proofing process are visible.
-4. All digital verification of evidence (e.g., via chip or wireless technologies) SHALL be performed by scanners and sensors that are integrated into the solution and in the entire field of view of the camera and the remote, live operator.
-5. Collection of biometrics SHALL be done in such a way that ensures that the biometric is collected from the applicant, and not another individual. All biometric requirements in [SP 800-63B, Section 5.2.3 Biometric Considerations](sp800-63b.html/#biometric_use) apply.
+2. 2.	The CSP SHALL require all actions taken by the applicant during the enrollment and identity proofing process to an operator. are clearly visible and without doubt.
```
Comment (jimfenton) [inline sp800-63a/sec5_proofing.md]: Stray period? Suggest "The CSP SHALL require all actions taken by the applicant during the enrollment and identity proofing process to be clearly visible to the remote operator. The operator SHALL direct the applicant as required to remove any doubt in the proofing process."

File: sp800-63a/sec5_proofing.md [hunk 6]
```diff
@@ -145,7 +145,7 @@ Virtual in-person identity proofing and enrollment transaction SHALL meet the fo
 
 ### <a name="trustref"></a> 5.4.4. Trusted Referee Requirements
 
-The CSP MAY use trusted referees, such as notaries, legal guardians, conservators, persons with power attorney, or some other form of certified/approved individuals that can vouch for or act on behalf of the individual in accordance with applicable laws, regulations, or agency policy.  The CSP MAY allow an individual that has successfully completed identity proofing to act as a trusted referee for another individual.  The CSP MAY use a trusted referee for both remote and in-person processes.  
+The CSP MAY use trusted referees, such as notaries, legal guardians, medical professionals, conservators, persons with power attorney, or some other form of certified/approved individuals that can vouch for or act on behalf of the individual in accordance with applicable laws, regulations, or agency policy.  The CSP MAY allow an individual that has successfully completed identity proofing to act as a trusted referee for another individual.  The CSP MAY use a trusted referee for both remote and in-person processes.
```
Comment (jimfenton) [inline sp800-63a/sec5_proofing.md]: "power attorney" -> "power of attorney"

File: sp800-63a/sec4_ial.md [hunk 7]
```diff
@@ -1,255 +1,261 @@
...
-Biometric Collection|No|Yes|Yes|
-Security Controls|N/A|[[SP 800-53]](#SP800-53) Moderate Baseline (or equivalent)|[[SP 800-53]](#SP800-53) High Baseline (or equivalent)
-
-
+<a name="sec4"></a>
```
Comment (jimfenton) [inline sp800-63a/sec4_ial.md]: I don't know what happened here, but this commit changed all of the line terminators in the file from NL (Unix standard) to CR NL (DOS standard), which is why the diff looks funny. We should try to keep the line terminators as consistent as possible. I'm guessing that this is coming from your loaner laptop. I can push a commit to clean that up if you want.

**Comment by jimfenton** [comment]
Commit 3ff94f5 changes the line terminators on 800-63b/sec10_usability.md. Some lines already had terminator issues, but there are more now.

**Review by jimfenton — CHANGES_REQUESTED** [review]
It's very hard to review the sections where the line terminators changed because everything shows up as a difference. Let me know if you have a way to clean those up or would like me to do so.

File: sp800-63a/sec3_definitions.md [hunk 8]
```diff
@@ -136,9 +136,6 @@ See *Authenticator Secret*.
 #### Trust Anchor
 A public or symmetric key that is trusted because it is directly built into hardware or software, or securely provisioned via out-of-band means, rather than because it is vouched for by another trusted entity (e.g. in a public key certificate).
 
-#### Unverified Name
```
Comment (jimfenton) [inline sp800-63a/sec3_definitions.md:139]: Should "Verified name" also have been deleted? It's not used.

File: sp800-63a/sec5_proofing.md (hunk 1 above)
Comment (paul-grassi) [reply sp800-63a/sec5_proofing.md]: fixed.  May be a good idea to break up though.

File: sp800-63a/sec5_proofing.md (hunk 3 above)
Comment (paul-grassi) [reply sp800-63a/sec5_proofing.md]: Fixed, but I am not sure I want to go with kiosk as that is a 'for example'.  But I'm happy to move away from 'solution'.

**Comment by paul-grassi** [comment]
Thanks @jimfenton.  I think no more editing on a plane :)

Also, terminator issues have to be due to laptop and editor.  I think I won't edit the doc any more this week.

File: sp800-63a/sec3_definitions.md (hunk 8 above)
Comment (paul-grassi) [reply sp800-63a/sec3_definitions.md:139]: done.  I was trying just to fix dberigni issues, but knocked this one out too.

**Comment by jimfenton** [comment]
Don't let this get in the way of being productive. Now that I know this is happening, I'll just watch for non-standard terminators and commit changes to fix that.

In case you aren't aware, by default GitHub lets people with write access to the repo you're sending a pull request to commit changes to your branch prior to the merge. I can do that with the existing line terminator issues if that's OK with you.

**Comment by paul-grassi** [comment]
Sounds good to me. Thanks.

**Review by jimfenton — APPROVED** [review]

**Pull request merged**